src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5958,29 +5958,41 @@ namespace HVACVariableRefrigerantFlow {
 			return;
 		}
 
-		// Get result when DX coil is off
-		PartLoadRatio = 0.0;
+		// Get result when DX coil is operating at the minimum PLR (1E-20) if not otherwise specified
+		PartLoadRatio = VRFTU( VRFTUNum ).MinOperatingPLR;
```

Comment:
@rraustad If mfrs asked for a min PLR input, then I guess that's useful.  But if the fix here solve the issue, maybe it's not worth the trouble.